lua/cmp/utils/async.lua

```diff
@@ -58,8 +61,8 @@ async.throttle = function(fn, timeout)
         time = vim.loop.now()
```

Comment:
IMO, we should add `this.running = false` to `stop`.

Reply:
I agree. Just fixed that.